### PR TITLE
Integration with pip. Fixes #14.

### LINKED ISSUE #14: Support pip installation
The best way to package and distribute a python package is through pip. Currently, the API does not support that as it doesn't provide a `setup.py`
Packaging projects in python: https://packaging.python.org/tutorials/packaging-projects/

There are also other minor warnings like:
```
warning: the following paths have collided (e.g. case-sensitive paths on 
a case-insensitive filesystem) and only one from the same colliding group 
is in the working tree:

  'bfxapi/Client.py'
  'bfxapi/client.py'
  'bfxapi/models/Order.py'
  'bfxapi/models/order.py'
  'bfxapi/models/Position.py'
  'bfxapi/models/position.py'
  'bfxapi/models/Subscription.py'
  'bfxapi/models/subscription.py'
  'bfxapi/models/Trade.py'
  'bfxapi/models/trade.py'
  'bfxapi/models/Wallet.py'
  'bfxapi/models/wallet.py'
```

Repro:
- Execute `pip install git+https://github.com/bitfinexcom/bitfinex-api-py.git`

### PR DESCRIPTION
**Pull request by redknightlois**
You may need to change some of the details like the version and email, but for the rest it works.

### CODE REVIEW

**Comment by JacobPlaster** [comment]
Hi @redknightlois thanks for the PR! Could you please give a description of the problem and your solution. Thanks!

**Comment by redknightlois** [comment]
This is just boilerplate registration and dependencies under the `setup.py` for Pypy the common pythonic way to distribute packages.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+from io import open
+
+here = path.abspath(path.dirname(__file__))
+
+# Arguments marked as "Required" below must be included for upload to PyPI.
+# Fields marked as "Optional" may be commented out.
+
+setup(
+    name='pybitfinex',  # Required
+    version='0.0.1',  # Required
```
Comment (JacobPlaster) [inline setup.py]: bump to `1.0.0`

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# Python 3 only projects can skip this import
+from io import open
+
+here = path.abspath(path.dirname(__file__))
+
+# Arguments marked as "Required" below must be included for upload to PyPI.
+# Fields marked as "Optional" may be commented out.
+
+setup(
+    name='pybitfinex',  # Required
```
Comment (JacobPlaster) [inline setup.py]: pybitfinex -> bitfinex-api-py

Just to keep consistency accross our other clients

**Comment by JacobPlaster** [comment]
Also looks like `pylint --rcfile=pylint.rc bfxapi` is throwing some errors

**Comment by redknightlois** [comment]
Go ahead and take the file alone and close this PR. My idea was having the ball rolling with the pypy support. I can do it but not until monday.

**Comment by JacobPlaster** [comment]
Sure @redknightlois, included in this recently merged PR https://github.com/bitfinexcom/bitfinex-api-py/pull/25.

Thanks again for the work.

**Pull request closed without merging**